<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RunMacroDlg</class>
 <widget class="QDialog" name="RunMacroDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Macros Launcher</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="gbFilter">
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="title">
      <string>Filter</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="locale">
           <locale language="English" country="UnitedStates"/>
          </property>
          <property name="text">
           <string>Start date:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDateEdit" name="deStart">
          <property name="locale">
           <locale language="English" country="UnitedStates"/>
          </property>
          <property name="displayFormat">
           <string>dd.MM.yyyy</string>
          </property>
          <property name="date">
           <date>
            <year>2000</year>
            <month>1</month>
            <day>1</day>
           </date>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="locale">
           <locale language="English" country="UnitedStates"/>
          </property>
          <property name="text">
           <string>End date:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDateEdit" name="deEnd">
          <property name="locale">
           <locale language="English" country="UnitedStates"/>
          </property>
          <property name="displayFormat">
           <string>dd.MM.yyyy</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pbUpdate">
        <property name="locale">
         <locale language="English" country="UnitedStates"/>
        </property>
        <property name="text">
         <string>Refresh</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbCheckAll">
        <property name="locale">
         <locale language="English" country="UnitedStates"/>
        </property>
        <property name="text">
         <string>Check all</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbUncheckAll">
        <property name="locale">
         <locale language="English" country="UnitedStates"/>
        </property>
        <property name="text">
         <string>Uncheck all</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="locale">
         <locale language="English" country="UnitedStates"/>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="text">
      <string>Macros for vibration analysis (ASV archives)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="asvTableView">
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RunMacroDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RunMacroDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
